index.js

```diff
@@ -0,0 +1,98 @@
+
+const url = "https://api.github.com/repos/Elena-Nam/Open-API-Project";
+fetch(url)
+  .then(response => response.json())
+  .then(data => console.log(data))
+  .catch((e) => console.log(e));
```

Comment:
Unless there's something in the instructions I'm missing or I'm not understanding the project, I don't believe you need this GitHub API call.